src/EncompassRest/Loans/Enums/FreOfferingIdentifier.cs

```diff
@@ -0,0 +1,34 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum FreOfferingIdentifier
+    {
```

Comment:
numeric values to match string values (eg. 210 = 210)

Reply:
In case you didn't see I've done as you suggested.